tripal_chado/src/ChadoBuddy/ChadoBuddyRecord.php

```diff
@@ -20,9 +22,118 @@ class ChadoBuddyRecord {
   protected string $base_table;
 
   /**
-   * The name of the chado schema this record from retrieved from.
+   * The name of the chado schema this record was retrieved from.
    * @var string
    */
   protected string $schema_name;
 
+  /**
+   * An associative array where the keys are chado table column
+   * names and the values are chado table record values.
+   * @var array
+   */
+  protected array $values;
+
+
+  /**
+   * Sets the value of the base table.
```

Comment:
It might be good to be clear what you mean here by the base table. For example
> In this context the base table is the core table for this buddy. For example, the base table would be cvterm for the CvtermBuddy. A more complex example, is for the PropertyBuddy where the base table is feature for a property buddy adding properties to the feature table.

Reply:
This change has been added